engine/flink/management/src/it/scala/pl/touk/nussknacker/engine/management/streaming/FlinkStreamingProcessManagerSpec.scala

```diff
@@ -40,6 +40,17 @@ class FlinkStreamingProcessManagerSpec extends FunSuite with Matchers with Strea
     cancel(processId)
   }
 
+  ignore("continue on timeout exception during process deploy") {
```

Comment:
please add comment that it is manual test because is hard to make it automatic

Reply:
Done